crates/deltalake/Cargo.toml

```diff
@@ -24,8 +25,8 @@ default = ["arrow", "deltalake-core/parquet"]
 datafusion = ["deltalake-core/datafusion"]
 datafusion-ext = ["datafusion"]
 gcs = ["deltalake-core/gcs"]
-glue = ["deltalake-core/glue"]
-glue-native-tls = ["deltalake-core/glue-native-tls"]
+glue = ["deltalake-catalog-glue"]
+glue-native-tls = ["deltalake-catalog-glue/native-tls"]
```

Comment:
was just wondering if there maybe is a way to propagate this from the general native-tls feature, and if this would even be wanted?.

Reply:
I have not found a good way to propagate features, it's a little tedious book-keeping :cry:

Reply:
not entirely sure, but I found the [example](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features) below.

```toml
[dependencies]
serde = { version = "1.0.133", optional = true }
rgb = { version = "0.8.25", optional = true }

[features]
serde = ["dep:serde", "rgb?/serde"]
```

The docs there say.. 
> In this example, enabling the serde feature will enable the serde dependency. It will also enable the serde feature for the rgb dependency, but only if something else has enabled the rgb dependency.

So if I read that correctly, we may be able to add `"deltalake-catalog-glue?/native-tls"` to the `native-tls` feature and it only takes effect, if glue is enabled?

Reply:
Coming back to this, I think this is a good idea, but it will require a features rework in the metacarte since there we also have `s3-native-tls` for example. I'll tackle that in a follow up pull request